libledger/LedgerParam.cpp

```diff
@@ -427,6 +427,16 @@ void LedgerParam::initConsensusConfig(ptree const& pt)
     // if the consensus node id is invalid, throw InvalidConfiguration exception
     parsePublicKeyListOfSection(mutableConsensusParam().sealerList, pt, "consensus", "node.");
     std::stringstream nodeListMark;
+    // sort the sealerList
+    if (g_BCOSConfig.version() >= V2_9_0)

```

Comment:
Maybe no need use `lessThan`.

use `std::less<h512>` or use `std::sort(mutableConsensusParam().sealerList.begin(),
            mutableConsensusParam().sealerList.end())` only.

Reply:
yes, you are right, thanks for the suggestion!

Reply:
> Maybe no need use `lessThan`.
> 
> use `std::less<h512>` or use `std::sort(mutableConsensusParam().sealerList.begin(), mutableConsensusParam().sealerList.end())` only.

yes, you are right, thanks for the suggestion!